src/System.ServiceProcess.ServiceController/src/Configurations.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard-Windows_NT;
+      netcoreapp-Windows_NT;
```

Comment:
correct UWP isn't supporting these. On top of the win32 APIs it requires admin access for most of these actions.